Pod/Classes/VOKCoreDataManager.m

```diff
@@ -8,6 +8,8 @@
 #import "VOKCoreDataManager.h"
 #import "VOKCoreDataManagerInternalMacros.h"
 
+#import <objc/runtime.h>
```

Comment:
Looks like this import can be removed from `NSManagedObject+VOKManagedObjectAdditions.m`. I can't comment on the line over there because it's not part of the diff.